integration-tests/swiss-knife/src/main/blueprint/blueprint.conf

```diff
@@ -0,0 +1,18 @@
+blueprint {
+  streamlets {
+    ingress = swissknife.SparkDataGenerator
```

Comment:
To limit the deployment size of this app a bit we cold change this to an Akka-based generator. that would safe at least 3 pods.